src/libraries/Common/tests/System/IO/PositionValueStream.cs

```diff
@@ -20,7 +20,7 @@ public override int Read(byte[] buffer, int offset, int count)
         {
             if (_remaining < 0)
             {
-                throw new ObjectDisposedException(typeof(PositionValueStream).Name);
+                ObjectDisposedException.Throw(typeof(PositionValueStream));
```

Comment:
```suggestion
                ObjectDisposedException.Throw(this);
```

Reply:
Moving a compile-time determined value to a runtime call to GetType()?  How un-Toub-like :smile: